tools/db_crashtest.py

```diff
@@ -179,7 +179,7 @@
     "max_key_len": 3,
     "key_len_percent_dist": "1,30,69",
     "read_fault_one_in": lambda: random.choice([0, 32, 1000]),
-    "write_fault_one_in": 0,
+    "write_fault_one_in": lambda: random.choice([0, 32, 1000]),
```

Comment:
I feel setting it to 32 may not allow a large compaction to finish, is this intended?

Reply:
Didn't see any issues when running locally, but I had disabled read fault injection. With both enabled, maybe this is too aggressive. I'll increase it and let's see how it goes.